db/migrate/20120222184238_create_slides.rb

```diff
@@ -1,4 +1,4 @@
-class CreateSlides < SolidusSupport::Migration[4.2]
+class CreateSlides < ActiveRecord::Migration[4.2]
```

Comment:
is this compatible with rails 5? `Migration[4.2]`

Reply:
https://guides.solidus.io/developers/extensions/testing-extensions.html
![image](https://user-images.githubusercontent.com/4744682/57558515-90272b80-7343-11e9-96c4-9c4c2da10c6f.png)